src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
```

Comment:
As we have a parameterized constructor it can be used here.
Create local variables for better readability such as:
String randomColor = colorSupplier.getRandomColor();
Then we can directly return new Ball(randomColor, randomNumber) from the method